scripts/qa_crowbarsetup.sh

```diff
@@ -3810,6 +3819,33 @@ function oncontroller_testsetup()
     nova stop "$instanceid"
     wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
 
+    # run tests for Magnum bay deployment
```

Comment:
@toabctl : I mean that right now, tempest tests are not running tests for `opensuse` driver in Magnum. And this part is covering them.